js/apps/admin-ui/cypress/e2e/clients_test.spec.ts

```diff
@@ -28,7 +28,7 @@ const createClientPage = new CreateClientPage();
 const clientDetailsPage = new ClientDetailsPage();
 const commonPage = new CommonPage();
 const listingPage = new ListingPage();
-const attributesTab = new AttributesTab();
+const attributesTab = new AttributesTab("client");
```

Comment:
Looking at the constructor of this class it seems this is not required, as the condition only checks for `"user"`. Let's remove this and make the argument optional in the class itself.